### PR TITLE
Added de_DE Translation. Corrected doube palette names on settings page

### PR DESCRIPTION
**Pull request by JustFxDev**
Hi aljawaid,

i corrected minore things:
The palette names on the settings page were double printed:

grafik

Added locale for de_DE
Added a translation help file (Excel xls) in locale folder

Cheers, Fx

### CODE REVIEW

**Comment by aljawaid** [comment]
@JustFxDev about the translation help file.... I had made and included this in each plugin to make it easier for users to translate.... am I wrong?

https://github.com/aljawaid/ColorManager/blob/9da75bb98f74345e276d1b21ce4bdc61e3942ff4/Locale/translations-starter-template.php

**Comment by aljawaid** [comment]
> Hi aljawaid,
> 
> i corrected minore things: The palette names on the settings page were double printed:
> 
> grafik

you should be getting the title like this....
image

**Comment by JustFxDev** [comment]
Ah ok! Some icon is missing and i get the text from the "alt=" tag. But why is it missing...

I doublecheck and respond.

Cheers, Fx

**Comment by JustFxDev** [comment]
I found the errorr and restored the icon include. A simple dot was missing in all four lines 

```
"/plugins/..."

"./plugins"..."
```

**Comment by JustFxDev** [comment]
Regarding the Excel Sheet:

You can copy the first column complete to deepl.com, get the translation from deepl and copy that result into the second column.

After that you can copy the complet php code for the new translations.php file in the 3rd column by simply cut and paste that.

I have to explain that further in the repo of that Excel File.

**Comment by aljawaid** [comment]
> Regarding the Excel Sheet:
> 
> You can copy the first column complete to deepl.com, get the translation from deepl and copy that result into the second column. Review the translation. Improve that. Ready.
> 
> After that you can copy the complete php code for the new translations.php file in the 3rd column by simply cut and paste that.
> 
> I have to explain that further in the repo of that Excel File.

**but you dont need to do any of that at all**.  Just enter your language strings in my starter template and move the file to your specific language folder.  

```
<?php
return array(
  //
  // GENERAL
  //
  'A new way to utilise colors to their full potential in Kanboard. Choose from 3 color palettes, default, extra (includes white) or create your own. Across all palettes, choose from over 50 colors to match your workflow across the interface.' => '',
  //
  // ColorManagerContoller.php
  //
  'Settings' => '',
  'Color Manager' => 'ITS ALL DONE FOR YOU ALREADY, JUST ENTER YOUR LANGUAGE STRINGS IN EACH LINE',
  'color-manager' => '',
  'Custom Colors' => '',
  'Add Color' => '',
  //
  // config/add_custom_color.php
  //
  'Color Name' => '',
  'Background Color' => '',
  'Border Color' => '',
  'Save Color' => '',
  //
  // config/colors.php
  //
  'Available Colors' => '',
```

**Comment by JustFxDev** [comment]
> **but you dont need to do any of that at all**. Just enter your language strings in my starter template and move the file to your specific language folder.

I know that. But it is more comfortable with deepl.com:

i copy just the text from (which one has to enter into that column but only once for all target languages):

grafik

I copy that to deepl.com (left box of the website)


grafik

and i get immediately the translation on the right side (with CRLF!)

grafik

and can copy that to 

grafik

Rework

and copy the automatic created rows from the yellow box to me new translations.php

grafik


done.

**Review by aljawaid — APPROVED** [review]

File: Template/config/colors.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 <div class="color-manager-page-header">
     <h2 class="">
-        <img class="color-manager-section-icon" src="/plugins/ColorManager/Assets/colour-palette.svg" alt="<?= t('Default Color Palette') ?>">
+        <img class="color-manager-section-icon" src="./plugins/ColorManager/Assets/colour-palette.svg" alt="<?= t('Default Color Palette') ?>">
```
[the remark this answers is not in the document]
Comment (aljawaid) [reply Template/config/colors.php:10]: Fixed in #23

**Pull request merged**